documentation/1.0.0/native/envs/interface_env

```diff
@@ -1,18 +1,19 @@
-APPLICATION_PORT=3569
-INSTALLED_PACKAGES=elevate-user elevate-scheduler elevate-project elevate-entity-management elevate-survey
-NOTIFICATION_SERVICE_BASE_URL=http://localhost:3002
-RATE_LIMITER_ENABLED=false
-RATE_LIMITER_GENERAL_LIMIT=50
-RATE_LIMITER_NUMBER_OF_PROXIES=3
-RATE_LIMITER_PUBLIC_LOW_LIMIT=5
-REQUIRED_PACKAGES=elevate-user@1.1.77 elevate-scheduler@1.0.4 elevate-project@1.1.10 elevate-entity-management@1.0.7 elevate-survey@1.0.21
-SCHEDULER_SERVICE_BASE_URL=http://localhost:4000
-SUPPORTED_HTTP_TYPES=GET POST PUT PATCH DELETE
-USER_SERVICE_BASE_URL=http://localhost:3001
-ALLOWED_HOST=http://localhost:8100
-PROJECT_SERVICE_BASE_URL=http://localhost:5000
-ENTITY_SERVICE_BASE_URL=http://localhost:5001
-REQUIRED_BASE_PACKAGES=user,notification,scheduler,project,entity,survey
-ROUTE_CONFIG_JSON_URLS_PATHS="https://raw.githubusercontent.com/ELEVATE-Project/user/staging/src/constants/interface-routes/configs.json,https://raw.githubusercontent.com/ELEVATE-Project/project-service/main/constants/interface-routes/configs.json,https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/main/constants/interface-routes/configs.json,https://raw.githubusercontent.com/ELEVATE-Project/scheduler/develop/src/constants/interface-routes/configs.json,https://raw.githubusercontent.com/ELEVATE-Project/entity-management/main/src/constants/interface-routes/configs.json"
\ No newline at end of file
+API_DOC_URL= /interface/api-doc
+APPLICATION_ENV= development
+APPLICATION_PORT= 3569
+ENTITY_SERVICE_BASE_URL= http://localhost:5001
+INSTALLED_PACKAGES= elevate-user elevate-mentoring elevate-scheduler elevate-project elevate-entity-management elevate-self-creation-portal elevate-survey
+NOTIFICATION_SERVICE_BASE_URL= http://localhost:3002
+PROJECT_SERVICE_BASE_URL= http://localhost:5000
+RATE_LIMITER_ENABLED= false
+RATE_LIMITER_GENERAL_LIMIT= 50
+RATE_LIMITER_NUMBER_OF_PROXIES= 3
+RATE_LIMITER_PUBLIC_LOW_LIMIT= 5
+REQUIRED_BASE_PACKAGES= user,notification,scheduler,project,entity-management,self-creation-portal,survey
+REQUIRED_PACKAGES="elevate-user@1.1.77 elevate-scheduler@1.0.4 elevate-project@1.1.16 elevate-entity-management@1.0.10 elevate-survey@1.0.27"
+ROUTE_CONFIG_JSON_URLS_PATHS="https://raw.githubusercontent.com/ELEVATE-Project/user/staging/src/constants/interface-routes/configs.json,https://raw.githubusercontent.com/ELEVATE-Project/project-service/main/constants/interface-routes/configs.json,https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/main/constants/interface-routes/configs.json,https://raw.githubusercontent.com/ELEVATE-Project/scheduler/develop/src/constants/interface-routes/configs.json,https://raw.githubusercontent.com/ELEVATE-Project/entity-management/main/src/constants/interface-routes/configs.json"
+SCHEDULER_SERVICE_BASE_URL= http://localhost:4000
+SELF_CREATION-PORTAL_SERVICE_BASE_URL= http://localhost:6001
+SUPPORTED_HTTP_TYPES= GET POST PUT PATCH DELETE
+SURVEY_SERVICE_BASE_URL= http://localhost:5007
```

Comment:
We don't need survey service here @MallanagoudaB , check docker setup env of interface to get an idea